wd_demo.au3

```diff
@@ -339,6 +339,10 @@ Func DemoElements()
 	Local $sElement, $aElements, $sValue, $sButton, $sResponse, $bDecode, $hFileOpen
 
 	_WD_Navigate($sSession, "https://google.com")
+	_WD_LoadWait($sSession, Default, 1000 * 10)
+
+	; Check if body DIV element is visible, as it can be hide all sub elements in case when COOKIE aproval message is visible
+	_WD_WaitElement($sSession, $_WD_LOCATOR_ByXPath, '//body/div[1][@aria-hidden="true"]', 0, 1000 * 60, $_WD_OPTION_NoMatch)
```

Comment:
> 1. Please explain why this is needed. Does this DIV element go away on its own or does it require user intervention?

In other way user may not be able to click on the confirmation message, I mean not able to click before script goes further.

> 2. "can be hide" isn't proper English. I think you meant "can hide".

Thanks will fix 

> 3. Should these changes to wd_demo be in a different PR?

yes it can be separate PR even mor there is a need to do the same for all other DEMOS so I plan to add additional Internal function.

Reply:
> > 2. "can be hide" isn't proper English. I think you meant "can hide".
> 
> Thanks will fix

Take a look: [small changes](https://github.com/Danp2/au3WebDriver/pull/292/commits/4500ffa6b0eace24fcdf0c6784d9c3da1880972b)

Reply:
> as he can hide

Not sure where "he" came from. Should be "as it can hide"

Reply:
> separate PR

I said different, not separate. There's an existing PR for wd_demo review, so why not put these wd_demo changes there?

Reply:
>> Does this DIV element go away on its own or does it require user intervention?

> In other way user may not be able to click on the confirmation message, I mean not able to click before script goes further.

To be sure I understand, you are saying that manual user intervention is required?

Reply:
> you are saying that manual user intervention is required?

Exactly.

Reply:
Can we treat it as aborted ?

Reply:
If _WD_WaitElement returns an error, then I would send a message to the log and bypass the rest of DemoElements to avoid a bunch of additional errors.

Reply:
> yes it can be separate PR .... even more there is a need to do the same for all other DEMOS so I plan to add additional Internal function.

Should I split it?

Reply:
I would move an wd_demo changes to their own PR

Reply:
take a look: 
https://github.com/Danp2/au3WebDriver/pull/293